### PR TITLE
Dt 1550 additional events

### PR DESCRIPTION
**Pull request by dnyanesh-sonwane**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ The specific events currently being raised are
 *   **OFFENDER_REGISTRATION_DELETED** is raised when an offender's registration is deleted. sourceId of the event is source id of the registration.
 *   **OFFENDER_REGISTRATION_DEREGISTERED** is raised when an offender's registration is deregistered. sourceId of the event is source id of the deregistation.
 *   **OFFENDER_MANAGEMENT_TIER_CALCULATION_REQUIRED** is raised when an offender's management tier needs recalculating. sourceId of the event is source id of the MANAGEMENT_TIER_EVENT.
+*   **SENTENCE_ORDER_REQUIREMENT_CHANGED** is raised when an offender's sentence order requirement is changed. sourceId of the event is requirementId.
+*   **SENTENCE_CHANGED** is raised when an offender's sentence is changed. sourceId of the event is sentenceId.
+*   **OFFENDER_MERGED** is raised when an offender's merge history is changed. sourceId of the event is mergeHistoryId.
+*   **CONVICTION_CHANGED** is raised when an offender's conviction is changed. sourceId of the event is convictionId.
```
Comment (andymarke) [inline README.md:34]: I think you are missing the `OFFENDER_OGRS_ASSESSMENT_CHANGE ` event is the documentation which would be something like 
OFFENDER_OGRS_ASSESSMENT_CHANGE when an OGRS assessment is added or updated

File: src/test/kotlin/uk/gov/justice/digital/hmpps/offenderevents/integration/PollCommunityApiTest.kt [hunk 2]
```diff
@@ -268,11 +323,11 @@ class PollCommunityApiTest : IntegrationTestBase() {
       assertThat(getNextMessageOnTestQueue().MessageAttributes.eventType.Value).isEqualTo("OFFENDER_BANANAS_CHANGED")
     }
     @Test
-    internal fun `when source is DISPOSAL then only offender disposal event is raised `() {
+    internal fun `when source is DISPOSAL then setence changed event is raised `() {
```
Comment (andrewrlee) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/offenderevents/integration/PollCommunityApiTest.kt:326]: little typo in `sentence`

**Review by andymarke — APPROVED** [review]
LGTM except the missing documentation

File: README.md (hunk 1 above)
Comment (andrewrlee) [reply README.md:34]: could maybe also describe what OGRS means?

Comment (andymarke) [reply README.md:34]: @dnyanesh-sonwane it stands for Offender Group Reconviction Scale and is a score to calculate the chances of reoffending

File: src/test/kotlin/uk/gov/justice/digital/hmpps/offenderevents/integration/PollCommunityApiTest.kt (hunk 2 above)
Comment (dnyanesh-sonwane) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/offenderevents/integration/PollCommunityApiTest.kt]: done

File: README.md (hunk 1 above)
Comment (dnyanesh-sonwane) [reply README.md:34]: done

**Review by andymarke — COMMENTED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**